src/scss/_post-single.scss

```diff
@@ -0,0 +1,13 @@
+.featured-image {
+  display: block !important;
+  width: 100% !important;
+
+  text-align: center !important;
+
+  }
+  .featured-image img {
+  display: inline-block !important;
+  height: 500px !important;
+  width: 100% !important;
+  object-fit: cover;
+  }
```

Comment:
If built-in classes are used in the markup, these classes aren't needed (with maybe the exception of the height).

Reply:
it was removed.